### PR TITLE
added solution for homework 26.1 ItemDaoHibernateImpl

### PR DESCRIPTION
**Pull request by Valdes2**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/dao/hibernate/ItemDaoHibernateImpl.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            if(transaction != null) {
+                transaction.rollback();
+            }
+        }
+        return deletedItem;
+    }
+
+    @Override
+    public List<Item> getAll() {
+        List<Item> allItems = null;
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/dao/hibernate/ItemDaoHibernateImpl.java]:
```suggestion
        List<Item> allItems = new ArrayList();
```

File: src/main/java/mate/academy/internetshop/util/HibernateUtil.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            logger.error("Can`t create session factory", e);
+            throw new RuntimeException(e);
+        }
+    }
+
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/util/HibernateUtil.java]: remove one empty line

File: src/main/resources/hibernate.cfg.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">Valdes</property>
+        <property name="connection.password">1234</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">validate</property>
+
+        <mapping class="mate.academy.internetshop.model.Item"/>
+    </session-factory>
+
+</hibernate-configuration>
```
Comment (Sofasmile) [inline src/main/resources/hibernate.cfg.xml]: add a  blank line at the end of the file

**Review by Sofasmile — APPROVED** [review]

File: src/main/java/mate/academy/internetshop/dao/hibernate/ItemDaoHibernateImpl.java (hunk 1 above)
Comment (Valdes2) [reply src/main/java/mate/academy/internetshop/dao/hibernate/ItemDaoHibernateImpl.java]: fixed

File: src/main/java/mate/academy/internetshop/util/HibernateUtil.java (hunk 2 above)
Comment (Valdes2) [reply src/main/java/mate/academy/internetshop/util/HibernateUtil.java]: fixed

File: src/main/resources/hibernate.cfg.xml (hunk 3 above)
Comment (Valdes2) [reply src/main/resources/hibernate.cfg.xml]: fixed

**Pull request merged**